test/runtests.jl

```diff
@@ -297,6 +297,153 @@ end
+    @test autodiff(Reverse, f12, Active, Active(2.0),  Const(4))[1] == (0.0, nothing)
+    @test autodiff(Forward, f12, Duplicated(2.0, 1.0), Const(4))    == (0.0,)
+    println("Done 12")
+
+    f13(x) = muladd(x, 3, x)
+    @test autodiff(Reverse, f13, Active, Active(2.0))[1][1] == 4
+    @test autodiff(Forward, f13, Duplicated(2.0, 1.0))[1]   == 4
+    println("Done 13")
+
+    f14(x) = x * cmp(x, 3)
+    @test autodiff(Reverse, f14, Active, Active(2.0))[1][1] == -1
+    @test autodiff(Forward, f14, Duplicated(2.0, 1.0))[1]   == -1
+    println("Done 14")
+
+    f15(x) = hypot(x, 2x)
+    @test autodiff(Reverse, f15, Active, Active(2.0))[1][1] == sqrt(5)
+    @test autodiff(Forward, f15, Duplicated(2.0, 1.0))[1]   == sqrt(5)
+    println("Done 15")
+
+    f16(x) = evalpoly(2, (1, 2, x))
+    @test autodiff(Reverse, f16, Active, Active(3.0))[1][1] == 4
+    @test autodiff(Forward, f16, Duplicated(3.0, 1.0))[1]   == 4
+    println("Done 16")
+
+    f17(x) = @evalpoly(2, 1, 2, x)
+    @test autodiff(Reverse, f17, Active, Active(3.0))[1][1] == 4
+    @test autodiff(Forward, f17, Duplicated(3.0, 1.0))[1]   == 4
+    println("Done 17")
+
+    f18(x) = widemul(x, 5.0f0)
+    @test autodiff(Reverse, f18, Active, Active(2.0f0))[1][1] == 5
+    @test autodiff(Forward, f18, Duplicated(2.0f0, 1.0f0))[1] == 5
+    println("Done 18")
+
+    f19(x) = copysign(x, -x)
+    @test autodiff(Reverse, f19, Active, Active(2.0))[1][1] == -1
+    @test autodiff(Forward, f19, Duplicated(2.0, 1.0))[1]   == -1
+    println("Done 19")
+
+    f20(x) = sum([ifelse(i > 5, i, zero(i)) for i in [x, 2x, 3x, 4x]])
+    @test autodiff(Reverse, f20, Active, Active(2.0))[1][1] == 7
+    @test autodiff(Forward, f20, Duplicated(2.0, 1.0))[1]   == 7
+    println("Done 20")
+
+    function f21(x)
+        nt = (a=x, b=2x, c=3x)
+        return nt.c
+    end
+    @test autodiff(Reverse, f21, Active, Active(2.0))[1][1] == 3
+    @test autodiff(Forward, f21, Duplicated(2.0, 1.0))[1]   == 3
+    println("Done 21")
+
+    f22(x) = sum(fill(x, (3, 3)))
+    @test autodiff(Reverse, f22, Active, Active(2.0))[1][1] == 9
+    @test autodiff(Forward, f22, Duplicated(2.0, 1.0))[1]   == 9
+    println("Done 22")
+
+    function f23(x)
+        a = similar(rand(3, 3))
+        fill!(a, x)
+        return sum(a)
+    end
+    @test autodiff(Reverse, f23, Active, Active(2.0))[1][1] == 9
+    @test autodiff(Forward, f23, Duplicated(2.0, 1.0))[1]   == 9
+    println("Done 23")
+
+    function f24(x)
+        try
+            return 3x
+        catch
+            return 2x
+        end
+    end
+    @test autodiff(Reverse, f24, Active, Active(2.0))[1][1] == 3
+    @test autodiff(Forward, f24, Duplicated(2.0, 1.0))[1]   == 3
+    println("Done 24")
+
+    function f25(x)
+        try
+            sqrt(-1.0)
+            return 3x
+        catch
+            return 2x
+        end
+    end
+    @test autodiff(Reverse, f25, Active, Active(2.0))[1][1] == 2
```

Comment:
add the test_broken for 1.6?

Reply:
Done, unfortunately the `@test 1==1 broken=cond` syntax is not available on Julia 1.6 so we have to use an `if` and `@test_broken`.

By the way I can remove the print statements just before this is merged, they are for debugging.